examples/examples/z_sub_liveliness.rs

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+use async_std::task::sleep;
+use clap::{App, Arg};
+use futures::prelude::*;
+use futures::select;
+use std::convert::TryFrom;
+use std::time::Duration;
+use zenoh::config::Config;
+use zenoh::prelude::r#async::*;
+
+#[async_std::main]
+async fn main() {
+    // Initiate logging
+    env_logger::init();
+
+    let (config, key_expr) = parse_args();
+
+    println!("Opening session...");
+    let session = zenoh::open(config).res().await.unwrap();
+
+    println!("Declaring Liveliness Subscriber on '{}'...", &key_expr);
+
+    let subscriber = session
+        .liveliness()
+        .declare_subscriber(&key_expr)
+        .res()
+        .await
+        .unwrap();
+
+    println!("Enter 'q' to quit...");
+    let mut stdin = async_std::io::stdin();
+    let mut input = [0_u8];
+    loop {
+        select!(
+            sample = subscriber.recv_async() => {
+                let sample = sample.unwrap();
+                println!(">> [Subscriber] Received {} ('{}')",
```

Comment:
Should we print more informed message, e.g. `The ZenohID has just come alive/it is dead` depending wether it is a `PUT` or a `DEL`?